lib/3scale_toolbox/tasks/copy_pricingrules_task.rb

```diff
@@ -0,0 +1,48 @@
+module ThreeScaleToolbox
+  module Tasks
+    class CopyPricingRulesTask
+      include CopyTask
+      include Helper
+
+      def call
+        metrics_map = metrics_mapping(source.metrics, target.metrics)
+        plan_mapping = application_plan_mapping(source.plans, target.plans)
+        plan_mapping.each do |plan_id, target_plan|
+          pricingrules_source = source.remote.list_pricingrules_per_application_plan(plan_id)
```

Comment:
Why is PricingRule defined as `pricingrule` but ApplicationPlan as `application_plan` ?

Reply:
:man_shrugging: 

I will change